package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "fliplet-widget-chat",
+  "version": "2.0.0",
+  "description": "Documentation is available at https://developers.fliplet.com/API/components/chat.html",
+  "main": "build.js",
+  "scripts": {
+    "eslint:github-action": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Fliplet/fliplet-widget-chat.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Fliplet/fliplet-widget-chat/issues"
+  },
+  "homepage": "https://github.com/Fliplet/fliplet-widget-chat#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.14.0",
+    "eslint-config-fliplet": "^1.0.6"
```

Comment:
The latest version is `1.0.7`.

Reply:
See https://github.com/Fliplet/eslint-config-client/blob/master/index.js

Some of global values you defined in `.eslintrc` don't need to be defined because it's already in Fliplet's config file.